trie/utils/verkle.go

```diff
@@ -24,11 +24,13 @@ import (
 )
 
 const (
-	VersionLeafKey  = 0
-	BalanceLeafKey  = 1
-	NonceLeafKey    = 2
-	CodeHashLeafKey = 3
-	CodeSizeLeafKey = 4
+	BasicDataLeafKey = 0
+	CodeHashLeafKey  = 1
+
+	BasicDataVersionOffset  = 0
+	BasicDataCodeSizeOffset = 5
```

Comment:
need to change so that the endianness is kept

Reply:
https://github.com/gballet/go-ethereum/pull/469